pubspec.yaml

```diff
@@ -7,8 +7,8 @@ environment:
   sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
-  meta: ^1.3.0
+  meta: ^1.8.0
```

Comment:
We don't need the new annotations. At max I'd set it to [`1.5.0`](https://pub.dev/packages/meta/versions/1.5.0) which matches our min sdk `2.12.0`